products/jbrowse-desktop/src/declare.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@jbrowse/core/ui/RecentSessionCard'
```

Comment:
just curious, why is this needed?

Reply:
I moved packages/core/ui/StartScreen.tsx to products/jbrowse-desktop/src/StartScreen.tsx and correspondingly needed to move a declare.d.ts for this item there

Reply:
I did this because jbrowse-web already had it's own specialized StartScreen so I figured there was no need to have one in core